inc/authldap.class.php

```diff
@@ -158,6 +158,10 @@ function preconfig($type) {
       }
    }
 
+   /**
```

Comment:
What is te point to add an empty documentation block? See also documentation about inherited methods.